### DISCUSSION TITLE
关于商业使用

### DISCUSSION THREAD

**Discussion by cheng6563** [zuihou/zuihou-admin-cloud#109] [02.提问]
<html>
<body>

功能 | 开源免费版 | 个人学习版 | 企业商用版 | 备注
-- | -- | -- | -- | --
转卖、售卖、赠送源码、视频、文档给第三方 | 无 | 无 | 无 | 违反必究！！！

</body>
</html>

外包项目，开发完业务功能之后连着源代码一起卖给客户，这算售卖源码吗？

**Comment by zuihou** [comment]
![image](https://user-images.githubusercontent.com/8500410/134361994-21ee70c6-c0ac-4261-ab00-7875334f67f8.png)
 
 
允许将二次开发之后的源码提供给第三方。
